index.js

```diff
@@ -30,6 +30,229 @@ for (let i = 0; i < links.length; i += 1) {
+    when an unknown printer took a galley of type and scrambled it 1960s with the relea`,
+    mechanizations: {
+      mech0: 'Rubi on rails',
+      mech1: 'css',
+      mech2: 'JavaScript',
+      mech3: 'html',
+    },
+    linkL: '#',
+    linkS: 'https://github.com/JansueT',
+  },
+  {
+    id: 6,
+    image: 'Snapshoot Portfolio.png',
+    title: 'Multi-Post Stories Gain+Glory',
+    explantion: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+    industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type
+    and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and
+    typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+    when an unknown printer took a galley of type and scrambled it 1960s with the relea`,
+    mechanizations: {
+      mech0: 'Rubi on rails',
+      mech1: 'css',
+      mech2: 'JavaScript',
+      mech3: 'html',
+    },
+    linkL: '#',
+    linkS: 'https://github.com/JansueT',
+  },
+];
+
+const cards = () => {
+  const card = projects.map(
+    (item) => ` <div class='card'>
+            <div class='imgplacesholder'></div>
+            <div class='container'>
+                <h2 class='popup_engin engin' >
+                   ${item.title}
+                </h2>
+                <ul class='eylem'>
+                    <li class='irem'>
+                        <button type='button' class='btn1'>
+                        ${item.mechanizations.mech0}
+                        </button>
+                    </li>
+                    <li class='irem'>
+                        <button type='button' class='btn1'>
+                        ${item.mechanizations.mech1}
+                            </button>
+                        </li>
+                        <li class='irem'>
+                            <button type='button' class='btn1'>
+                            ${item.mechanizations.mech2}
+                            </button>
+                        </li>
+                        <li class='irem'>
+                            <button type='button' class='btn1'>
+                            ${item.mechanizations.mech3}
+                            </button>
+                        </li>
+                    </ul>
+                <button type='button' onclick='popupDetails(${item.id})'  class='btngreen'>
+                 See Project
+                </button>
+              </div>  
+            </div>`,
+  );
+  return card;
+};
+window.addEventListener('load', () => {
+  document.getElementById('cards_container').innerHTML = cards().join('');
+});
+const check1 = { title1: 'Keeping track of hundreds of components' };
+const popupContainer = document.getElementById('popup_window_container');
+// const poWindow = document.getElementById('popup_window');
+// eslint-disable-next-line no-unused-vars
```

Comment:
[OPTIONAL] Kindly remove unused code as this does not make your code neat.

[OPTIONAL] Also, kindly enable linters as this will make you commit to writing good and clean code.